examples/c/mass_spring_cond_solver.c

```diff
@@ -120,6 +121,7 @@ int main() {
             if (info->condensing_time < min_info.condensing_time) min_info.condensing_time = info->condensing_time;
             if (info->solve_QP_time < min_info.solve_QP_time) min_info.solve_QP_time = info->solve_QP_time;
             if (info->interface_time < min_info.interface_time) min_info.interface_time = info->interface_time;
+            if (info->num_iter < min_info.num_iter) min_info.num_iter = info->num_iter;
```

Comment:
the number of iterations should not change between repetitions. Instead of having the minimum, I would instead do an assert that for i>1, niter[i] = niter[i-1] to avoid having the solver warm started